### DISCUSSION TITLE
Text array field for tagging (similar to DropDown with allowed additions, but without dropdown) required

### DISCUSSION THREAD

**Discussion by mkrecek234** [fomantic/Fomantic-UI#1573] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Feature Request
I would like to implement
a) a simple text entry field for multiple email recipients. It should be designed identically to a Fomantic-UI dropdown field with allowed additions BUT without the dropdown. So just enter the email address, enter "return" and get a grey box with the mail plus an "X" delete icon. You can easily add further ones then.

Other uses for this kind of input type are free tagging inputs, like: "Holiday 2020 X      Sunset X      Family trip X"

b) same field type but for multiple file uploads: So you choose multiple files to upload, and each file is listed in the same manner as above under a).

Any idea how to easily implement either with existing components or anyone who can help implementing this?

## Screenshot (if possible)
![image](https://user-images.githubusercontent.com/28671486/87421808-d62aef80-c5d7-11ea-802e-df5ebd6dec72.png)

**Comment by lubber-de** [comment, marked as answer]
You should still use the dropdown module but don't provide any option values to it.
If you dont want the dropdown icon, then remove it via css.

See https://jsfiddle.net/lubber/m9srqbco/5/
![tag_dropdown](https://user-images.githubusercontent.com/18379884/87422660-50a83f00-c5d9-11ea-9eb1-9a12196f9409.gif)

**Comment by mkrecek234** [comment]
Great, thanks a lot for your hint. Looks perfect.
